### PR TITLE
Determine instrument tile color based on filters

### PR DESCRIPTION
**Pull request by jadalilleboe**
Determines color for the instrument tiles based on the filters, here's the color for these filters:
Screen Shot 2022-04-15 at 3 58 46 PM
And with just the first 3:
Screen Shot 2022-04-15 at 4 03 12 PM
Also I made the galaxy points green to distinguish from sources

### CODE REVIEW

**Comment by mcoughlin** [comment]
@jadalilleboe what happens if there are multiple filters requested per field? is it just the first one?

**Comment by jadalilleboe** [comment]
@mcoughlin it's a hash that combines all the filters into a string and then computes a hex code from that.

**Comment by mcoughlin** [comment]
@jadalilleboe i am kind of wondering if the ability to cycle through would make sense looking ahead to when we want to add multiple filters per field?

**Comment by jadalilleboe** [comment]
@mcoughlin what do you mean cycle through?

**Comment by mcoughlin** [comment]
Like I want to be able to add any field with any possible filter to it, so I think maybe one intuitive way to do it is by clicking and offering any set of filters. One way to do it would be changing color with every click, another would maybe be like piece of pie with each optional filter in the colors?

**Comment by jadalilleboe** [comment]
I'm gonna close this, I'll open a new PR if I start working on this.

**Comment by mcoughlin** [comment]
@jadalilleboe I think what you have now is better than before for sure, so I would think we should get this cleaned up and merged.

**Comment by jadalilleboe** [comment]
ok!

File: static/js/components/LocalizationPlot.jsx [hunk 1]
```diff
@@ -363,7 +380,7 @@ const GeoJSONGlobePlot = ({
           .data(galaxies.features)
           .enter()
           .append("circle")
-          .attr("fill", (d) => (visibleOnSphere(d) ? "red" : "none"))
+          .attr("fill", (d) => (visibleOnSphere(d) ? "green" : "none"))
```
Comment (mcoughlin) [inline static/js/components/LocalizationPlot.jsx:383]: This was just to differentiate galaxies from sources or some other reason?

**Review by mcoughlin — APPROVED** [review]
This looks nice @jadalilleboe

File: static/js/components/LocalizationPlot.jsx (hunk 1 above)
Comment (jadalilleboe) [reply static/js/components/LocalizationPlot.jsx:383]: yeah!

**Review by guynir42 — APPROVED** [review]
LGTM

**Pull request merged**